### PR TITLE
Update sensitivity-labels-sharepoint-onedrive-files.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Sharepoint latest version Info

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ For this preview, use the OneDrive sync app version 19.002.0121.0008 or later on
 
 ## Prepare the SharePoint Online Management Shell for the preview
 
-Before you enable the preview, ensure that you're running the latest SharePoint Online Management Shell. If you already have the latest version, you can go ahead and enable the preview.
+Before you enable the preview, ensure that you're running SharePoint Online Management Shell Version 16.0.19418.12000 or above. If you already have the latest version, you can go ahead and enable the preview.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
Before you enable the preview, ensure that you're running SharePoint Online Management Shell version 16.0.19418.12000 or above. If you already have the latest version, you can go ahead and enable the preview.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review - minor edit

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #715

Thank you.

**Comment by cabailey** [comment]
Approving this change

**Pull request merged**